pkg/eval/ievaluator.go

```diff
@@ -55,3 +55,18 @@ func (s *StateChangeNotification) ToMap() map[string]interface{} {
 		"flagKey": s.FlagKey,
 	}
 }
+
+func StateChangeNotificationsToMap(scns []StateChangeNotification) map[string]interface{} {
```

Comment:
it may be possible to build this map within the `json_evaluator.go` `Merge` method, rather than the existing `[]StateChangeNotification`

Reply:
done